assets/_scss/themes/_usa.scss

```diff
@@ -16,13 +17,13 @@ li {
 }
 
 a {
-  color: $color-main;
+  color: $color-primary;
```

Comment:
So I've been thinking about this:

How will people use the theme file or create new themes?

Whats to be gained by putting the color rules in the themes file? Like if I want to change a color for my site, I'd prefer to just set a variable, I won't need all this code here because it's just setting the color for various things. 

One worry I have here is that this file is already pretty large, and it's the begging of the other code, meaning this file could potentially get huge. It migiht be good to really think about what goes in the theme file, (I'm still thinking about it right now). I think variables themselves are the best way to control theme.